src/TestFramework/Coverage/Trace.php

```diff
@@ -60,6 +60,8 @@ public function getSourceFileInfo(): SplFileInfo;
      */
     public function getRealPath(): string;
 
+    public function getRelativePathname(): string;
```

Comment:
why is this needed? If its for the same reasons as `getRealPath` I would add a comment as well. Otherwise this is confusing.

Note to self: might be worth extracting it in a separate interface for clarity, which `Trace` could extend from

Reply:
`PathFilterIterator` requires it

Reply:
See https://github.com/infection/infection/pull/1241